NEWS

```diff
@@ -1,3 +1,23 @@
+openscap-1.3.2                                                  13-01-2020
```

Comment:
I recall that we discussed with the team long time ago that we want to make the release notes more exciting, but I don't know how to do that. If you have any idea, feel free to be creative.

Reply:
Mmm, I don't know. This update really feels like a maintenance release to me.